docs/source/quick_start.md

```diff
@@ -86,7 +86,7 @@ When enabling continuous batching, batch size should not be specified.
 Users can leverage multi-Qranium and other supported features along with continuous batching.
 
 ```bash
-python -m QEfficient.cloud.infer --model_name TinyLlama/TinyLlama_v1.1 --batch_size 3 --prompt_len 32 --ctx_len 128 --num_cores 16 --device_group [0] --prompt "My name is|The flat earth 
+python -m QEfficient.cloud.infer --model_name TinyLlama/TinyLlama_v1.1 --batch_size 1 --prompt_len 32 --ctx_len 128 --num_cores 16 --device_group [0] --prompt "My name is|The flat earth 
```

Comment:
Please remove `--batch_size` why should we tell users to pass two batch_sizes? It's confusing.